Tools/CISupport/ews-build/steps.py

```diff
@@ -728,7 +729,10 @@ def run(self):
             GIT_PASSWORD=access_token,
         )
 
-        return super().run()
+        rc = yield super().run()
+        if rc == SUCCESS and self.getProperty('buildername', '').lower() not in ('commit-queue', 'merge-queue', 'unsafe-merge-queue'):
```

Comment:
probably a good idea to separate out the list in a variable (so that it's easy for anyone to modify it in future). e.g.: QUEUES_WITH_PUSH_ACCESS